packages/big-design/src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { forwardRef, memo, Ref } from 'react';
+
+import { useUniqueId } from '../../hooks';
+import { Flex } from '../Flex';
+
+import { HiddenCheckbox, StyledSwitchLabel } from './styled';
+
+export type SwitchProps = React.InputHTMLAttributes<HTMLInputElement>;
+
+interface PrivateProps {
+  forwardedRef: Ref<HTMLInputElement>;
+}
+
+export const RawSwitch: React.FC<SwitchProps & PrivateProps> = ({ checked, disabled, forwardedRef, ...props }) => {
+  const uniqueCheckboxId = useUniqueId('switch');
+  const id = props.id ? props.id : uniqueCheckboxId;
```

Comment:
Do you think it should be called `uniqueCheckboxId` or `uniqueSwitchId`? I get why it would be called either one, but curious what you think about it.

Reply:
I didn't give it much thought earlier, but I feel better to update to `uniqueSwitchId` in order to avoid any confusion with checkbox side of things.